setup.cfg

```
@@ -41,10 +41,10 @@ docs =
 exclude = .git,__pycache__,docs/conf.py,build,dist
 
 [tool:pytest]
-testpaths = "tests" "docs" "exovetter"
-astropy_header = true
-open_files_ignore = kplr006922244-2010078095331_llc.fits
-doctest_plus = enabled
+#testpaths = "tests" "docs" "exovetter"
+#astropy_header = true
+#open_files_ignore = kplr006922244-2010078095331_llc.fits
+#doctest_plus = enabled
```

Why comment these out? These are set up on purpose to control `pytest` behavior.'
```suggestion
testpaths = "tests" "docs" "exovetter"
astropy_header = true
open_files_ignore = kplr006922244-2010078095331_llc.fits
doctest_plus = enabled
```

---

I can't run tests with these uncommented.  So we need a better solution.

---

Can you please elaborate? Cannot run the tests locally or on CI? What errors?

---

I can't run them locally. It causes pytest to throw errors.

================================================================= test session starts =================================================================
platform darwin -- Python 3.7.9, pytest-6.1.1, py-1.9.0, pluggy-0.13.1
rootdir: /Users/smullally/Python_Code/exovetter-repos/official/exovetter, configfile: setup.cfg
plugins: jupyter-server-1.0.5, tornasync-0.6.0.post2
collected 19 items                                                                                                                                    
INTERNALERROR> Traceback (most recent call last):
INTERNALERROR>   File "/Users/smullally/anaconda/envs/python37/lib/python3.7/site-packages/_pytest/main.py", line 257, in wrap_session
INTERNALERROR>     session.exitstatus = doit(config, session) or 0
INTERNALERROR>   File "/Users/smullally/anaconda/envs/python37/lib/python3.7/site-packages/_pytest/main.py", line 312, in _main
INTERNALERROR>     config.hook.pytest_collection(session=session)
INTERNALERROR>   File "/Users/smullally/anaconda/envs/python37/lib/python3.7/site-packages/pluggy/hooks.py", line 286, in __call__
INTERNALERROR>     return self._hookexec(self, self.get_hookimpls(), kwargs)
INTERNALERROR>   File "/Users/smullally/anaconda/envs/python37/lib/python3.7/site-packages/pluggy/manager.py", line 93, in _hookexec
INTERNALERROR>     return self._inner_hookexec(hook, methods, kwargs)
INTERNALERROR>   File "/Users/smullally/anaconda/envs/python37/lib/python3.7/site-packages/pluggy/manager.py", line 87, in <lambda>
INTERNALERROR>     firstresult=hook.spec.opts.get("firstresult") if hook.spec else False,
INTERNALERROR>   File "/Users/smullally/anaconda/envs/python37/lib/python3.7/site-packages/pluggy/callers.py", line 203, in _multicall
INTERNALERROR>     gen.send(outcome)
INTERNALERROR>   File "/Users/smullally/anaconda/envs/python37/lib/python3.7/site-packages/_pytest/config/__init__.py", line 1207, in pytest_collection
INTERNALERROR>     self._validate_config_options()
INTERNALERROR>   File "/Users/smullally/anaconda/envs/python37/lib/python3.7/site-packages/_pytest/config/__init__.py", line 1230, in _validate_config_options
INTERNALERROR>     self._warn_or_fail_if_strict("Unknown config option: {}\n".format(key))
INTERNALERROR>   File "/Users/smullally/anaconda/envs/python37/lib/python3.7/site-packages/_pytest/config/__init__.py", line 1266, in _warn_or_fail_if_strict
INTERNALERROR>     self.issue_config_time_warning(PytestConfigWarning(message), stacklevel=3)
INTERNALERROR>   File "/Users/smullally/anaconda/envs/python37/lib/python3.7/site-packages/_pytest/config/__init__.py", line 1318, in issue_config_time_warning
INTERNALERROR>     warnings.warn(warning, stacklevel=stacklevel)
INTERNALERROR> pytest.PytestConfigWarning: Unknown config option: astropy_header

---

`pip install .[test]`

---

pip install -e .[test] seemed to work.